docker/centos8/Dockerfile

```diff
@@ -1,13 +1,17 @@
-FROM centos:centos8
-
-USER 0
-
-# Package installation
-RUN yum update -y
-
-## Common packages for linux build environment
-RUN yum -y group install "Development Tools"
-RUN yum install -y curl libcurl-devel zlib-devel clang python38 pkg-config git bzip2 unzip make wget sudo cmake
-RUN yum --enablerepo=powertools install -y zlib-static gmock gtest
-
+FROM centos:centos8
+
+USER 0
+
+
+RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-Linux-* &&\
+    sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-Linux-*

```

Comment:
What is wrong with installing packages from `mirror`? Or do we need old/specific versions from `vault`

Reply:
it failed to build the docker image - not sure what is the cause of issue

Reply:
Ok probably because CentOS8 has reached end of lifecycle, and the repos are not more available, so need to switch to `vault` for archives. Should be fine.